### PR TITLE
Update ReadMe and setup benchmarks

### PR DESCRIPTION
**Pull request by serge-sans-paille**
@cbeck88 and @jaydee-io did I miss something obvious in the ebnchmarks? Does that looks like a fair enough ground?

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1 +1,2 @@
 serge-sans-paille <sguelton@quarkslab.com>
+Jérôme Dumesnil
```
Comment (jaydee-io) [inline AUTHORS]: Thanks ;-) ! You can also add my mail <jerome.dumesnil@gmail.com> if you want to.

File: README.rst [hunk 2]
```diff
@@ -23,7 +31,8 @@ Requirements
 ------------
 
 A C++ compiler that supports C++14. Clang version 5 is a good pick, GCC version
```
Comment (jaydee-io) [inline README.rst]: Maybe a section about frozen testing (How to build/run tests, C+17 / GoogleBenchmark need, ...) could be added to the README ?

**Comment by serge-sans-paille-qb** [comment]
@jaydee-io and @cbeck88 you can also review this blogpost: https://public.etherpad-mozilla.org/p/frozen-02 that will cover the release; You're both cited ;-)

File: README.rst (hunk 2 above)
Comment (serge-sans-paille-qb) [reply README.rst]: Correct! will do.

**Comment by serge-sans-paille-qb** [comment]
@jaydee-io I prefer volatile because its semantic is well defined by the sandard :-)

File: AUTHORS (hunk 1 above)
Comment (serge-sans-paille-qb) [reply AUTHORS]: done (and for @cbeck88 too btw)

File: README.rst [hunk 3]
```diff
@@ -128,16 +137,29 @@ in the following:
 
     constexpr frozen::unordered_set<frozen::string, 2, olaf/*custom hash*/> hans = { "a", "b" };
 
+Tests and Benchmarks
+--------------------
+
+Using hand-written Makefiles carfted with love and care:
```
Comment (smkanadl) [inline README.rst]: I think it is 'crafted'... :-)

**Comment by serge-sans-paille-qb** [comment]
Updated, thanks!

**Pull request merged**